nntrainer/tensor/manager.cpp

```diff
@@ -409,6 +416,28 @@ Manager::requestInputs(const GraphNode &node,
         var_name + Var_Grad::grad_suffix, /// name
         Tensor::Initializer::ZEROS        /// tensor initializer
       );
+    } else {
+      /** requesting externally allocated tensor for input */
+      var = tensor_pool.requestExternallyAllocateTensor(
+        dim,                      /// tensor dim
+        var_name,                 /// name
+        Tensor::Initializer::NONE /// tensor initializer
+      );
+
+#ifdef ENABLE_TEST
+      grad = tensor_pool.requestTensor(
+        dim, /// tensor dim
+        grad_exec_order, grad_ls,
+        var_name + Var_Grad::grad_suffix, /// name
+        Tensor::Initializer::ZEROS        /// tensor initializer
+      );
+#else
+      grad = tensor_pool.requestExternallyAllocateTensor(
+        dim,                              /// tensor dim
+        var_name + Var_Grad::grad_suffix, /// name
+        Tensor::Initializer::ZEROS        /// tensor initializer
+      );
+#endif
     }
 
     /**
```

Comment:
Wonder if this is fixed?

Reply:
can you please check the link - its does not point to the selected change.

Reply:
The line is right below this comment. (unable to comment on that line)
https://github.com/nnstreamer/nntrainer/blob/f2ab7f8b8cce3eaae296f22017138386fab6e6eb/nntrainer/tensor/manager.cpp#L444-L446

Reply:
done.